mod-orders/schemas/composite_po_line.json

```diff
@@ -209,8 +209,13 @@
     },
     "source": {
       "description": "details related to the source of this purchase order line",
-      "type": "object",
-      "$ref": "../../mod-orders-storage/schemas/source.json"
+      "type": "string",
+      "enum": [
```

Comment:
Hi @KVupp,
since the source enum locates in both composite_po_line.json and po_line.json schemas, maybe it may make sense to store this enum in a separate file and only refer to it as it was initially for source.json?

Reply:
except for receipt_status, we are maintaining enum in the same file as it is used, just tried to keep it uniform, I can change it if we see any advantage of moving it to a different file

Reply:
It seemed to me making sense to keep just a reference to the entity which is encountered in different places. But it's up to you. I'm fine with current version. Thanks!

Reply:
Changed it to use a common file , to limit changes in multiple places